resources/js/Components/Listboxes/ProjectGroupListbox.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import {Listbox, ListboxButton, ListboxOption, ListboxOptions} from "@headlessui/vue";
+
+import {ChevronDownIcon} from "@heroicons/vue/outline";
+import Button from "@/Jetstream/Button.vue";
+import {CheckIcon} from "@heroicons/vue/solid";
+import {ref} from "vue";
+
+const props = defineProps({
+    projectGroups: {
+        type: Object,
+        required: true
+    },
+    selectedProjectGroup: {
+        type: Object,
+        required: false
+    }
+})
+
+const currentGroup = ref(props.selectedProjectGroup ? props.selectedProjectGroup : null)
+
+const emit = defineEmits(['update:selectedProjectGroup'])
+
+</script>
+
+<template>
+    <Listbox as="div" v-model="currentGroup" :on-update:model-value="$emit('update:selectedProjectGroup', currentGroup)" id="room">
+        <ListboxButton class="inputMain w-full h-10 cursor-pointer truncate flex p-2">
+            <div class="flex-grow flex text-left xsDark">
+                {{
+                    currentGroup ? currentGroup.name : $t('Search project group')
```

Comment:
COULD: Immer besser das in funktionen auszulagern im script setup, dadurch ist das template sauber und bläht sich nicht auf dauer so auf